builder/amazon/chroot/builder.go

```diff
@@ -180,6 +185,16 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 				errs, errors.New("If root_device_name is specified, ami_block_device_mappings must be specified"))
 		}
 	}
+	valid := false
+	for _, validArch := range []string{"x86_64", "arm64"} {
```

Comment:
just for the info: https://godoc.org/sort#SearchStrings could save us a few lines here 🙂

Reply:
Also, amazon code has:
```
const (
	// ArchitectureValuesI386 is a ArchitectureValues enum value
	ArchitectureValuesI386 = "i386"

	// ArchitectureValuesX8664 is a ArchitectureValues enum value
	ArchitectureValuesX8664 = "x86_64"

	// ArchitectureValuesArm64 is a ArchitectureValues enum value
	ArchitectureValuesArm64 = "arm64"
)
```

Do we want to allow for `i386` as well ?